src/ui/components/results/paginationcomponent.js

```diff
@@ -164,7 +164,13 @@ export default class PaginationComponent extends Component {
   }
 
   updatePage (offset) {
-    this._onPaginate();
+    const results = this.core.globalStorage.getState(StorageKeys.VERTICAL_RESULTS) || {};
+    let oldOffset = this.core.globalStorage.getState(StorageKeys.SEARCH_OFFSET) || 0;
+    const limit = this.core.globalStorage.getState(StorageKeys.SEARCH_CONFIG).limit;
```

Comment:
are we guaranteed to get a search config?

Reply:
Everything should default to an empty object if it hasn't been set, so this should be all right.

Reply:
We are guaranteed to get a SearchConfig. A default one is created if not supplied to Answers.init. The limit in this default SearchConfig will be 20.